mlflow/server/js/scripts/package-scripts.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
```

Comment:
This file looks like it should have some edge blocking, wdyt?

Reply:
SGTM. Should we edge-block commands like `databricks-webpack start`?

Reply:
Yeah cause that doesn't exist here; not the biggest deal though if this is not easy

Reply:
I think it's fine to include them because we don't touch them on OSS.